azure/table_azure_application_insight.go

```diff
@@ -0,0 +1,218 @@
+package azure
+
+import (
+	"context"
+
+	"github.com/Azure/azure-sdk-for-go/services/appinsights/mgmt/2020-02-02/insights"
+	"github.com/turbot/steampipe-plugin-sdk/v5/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin/transform"
+
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin"
+)
+
+//// TABLE DEFINITION ////
+
+func tableAzureApplicationInsight(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "azure_application_insight",
+		Description: "Azure Application Insight",
+		Get: &plugin.GetConfig{
+			KeyColumns: plugin.AllColumns([]string{"name", "resource_group"}),
+			Hydrate:    getApplicationInsight,
+			IgnoreConfig: &plugin.IgnoreConfig{
+				ShouldIgnoreErrorFunc: isNotFoundError([]string{"ResourceNotFound", "ResourceGroupNotFound", "404"}),
+			},
+		},
+		List: &plugin.ListConfig{
+			Hydrate: listApplicationInsights,
+		},
+		Columns: azureColumns([]*plugin.Column{
+			{
+				Name:        "name",
+				Type:        proto.ColumnType_STRING,
+				Description: "The friendly name that identifies the application insight",
+			},
+			{
+				Name:        "id",
+				Description: "Contains ID to identify the application insight uniquely",
```

Comment:
```suggestion
				Description: "Contains ID to identify the application insight uniquely.",
```
@karanpopat could you please add the missing `.` at the end of column descriptions?